pkg/controller/installation/validation.go

```diff
@@ -315,6 +315,11 @@ func validateCustomResource(instance *operatorv1.Installation) error {
 			instance.Spec.FlexVolumePath)
 	}
 
+	// Verify that the volumePlugin value is valid - either "Enabled" or "Disabled"
+	if instance.Spec.VolumePlugin != "Enabled" && instance.Spec.VolumePlugin != "Disabled" && instance.Spec.VolumePlugin != "" {
```

Comment:
I'm hoping we can move this validation into the CRD definition by defining a type for VolumePlugin with only Enabled and Disabled as the two valid values.

Reply:
Will do.